packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -148,8 +149,17 @@ const App = (props) => {
         history.push(path)
     }
 
+    // Storefront Preview script. Your storefront needs to load this script for Storefront Preview feature
+    const storefrontPreviewClientScript =
+        process.env.NODE_ENV === 'development'
+            ? 'http://localhost:4000/mobify/bundle/development/static/storefront-preview.js'
```

Comment:
Customers will not run Runtime Admin on localhost:4000. Instead, I'd only use the Runtime Admin production URL.

Reply:
We still use localhost:4000, though! What's the right balance for customer bundle size vs internal developer convenience?

Reply:
@wjhsf maybe moving this into the sdk is a better way.

Reply:
That's where the code will live, but we have to decide what code to put there. Any code that's purely for internal development is an unnecessary performance hit for shoppers.